contracts/libs/Tx.sol

```diff
@@ -809,4 +828,33 @@ library Tx {
                 abi.encodePacked(BURN_CONCENT, _tx.fromIndex, nonce, _tx.amount)
             );
     }
+
```

Comment:
Should add tests in `txSerialization.test.ts` for `massMigration_decode` and `massMigration_size`.